src/draft-service.ts

```diff
@@ -559,31 +559,13 @@ async function syncDrafts({
       localDraftsStorage = await getLocalDrafts()
       localDraftsStorage.deletedFormSubmissionDrafts =
         newDeletedFormSubmissionDrafts
-      await setDrafts(localDraftsStorage)
+      await setAndBroadcastDrafts(localDraftsStorage)
     }
 
-    localDraftsStorage.syncedFormSubmissionDrafts =
-      await getFormSubmissionDrafts(formsAppId, abortSignal)
-    await setDrafts(localDraftsStorage)
-
-    console.log(
-      'Ensuring all draft data is available for offline use for synced drafts',
-      localDraftsStorage.syncedFormSubmissionDrafts,
-    )
-    if (localDraftsStorage.syncedFormSubmissionDrafts.length) {
-      for (const formSubmissionDraft of localDraftsStorage.syncedFormSubmissionDrafts) {
-        await getDraftSubmission(formSubmissionDraft, abortSignal).catch(
-          (error) => {
-            console.warn('Could not download Draft Data as JSON', error)
-          },
-        )
-      }
-    }
-
-    console.log(
-      `Attempting to upload ${localDraftsStorage.unsyncedDraftSubmissions.length} local unsynced drafts(s).`,
-    )
     if (localDraftsStorage.unsyncedDraftSubmissions.length) {
+      console.log(
+        `Attempting to upload ${localDraftsStorage.unsyncedDraftSubmissions.length} local unsynced drafts(s).`,
+      )
```

Comment:
Can we move this back outside to the `if`, or move the other two logs `Removing local draft data for deleted drafts` and `Ensuring all draft data is available for offline use for synced drafts` inside the corresponding `if`s.